src/test/docker/bionic/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fluxrm/flux-core:bionic
+FROM fluxrm/flux-core:bionic-v0.27.0
```

Comment:
Update to v0.28.0 tag?

Reply:
Yup, right on time with flux-core! Just pushed up a change that updates this to `v0.28.0`.